conan/tools/build/flags.py

```diff
@@ -198,11 +198,15 @@ def _cppstd_msvc(visual_version, cppstd):
     if visual_version >= "191":
         v17 = "c++17"
         v20 = "c++latest"
-    if visual_version >= "193":
+    if visual_version >= "192":
         v20 = "c++20"
+    if visual_version >= "193":
         v23 = "c++latest"
 
-    flag = {"14": v14, "17": v17, "20": v20, "23": v23}.get(str(cppstd), None)
+    return {"14": v14, "17": v17, "20": v20, "23": v23}.get(str(cppstd), None)
+
+def _cppstd_msvc(visual_version, cppstd):
+    flag = cppstd_msvc_flag(visual_version, cppstd).get(str(cppstd), None)
```

Comment:
It seems this is redundant?
```suggestion
    flag = cppstd_msvc_flag(visual_version, cppstd)
```